docs/content/en/docs/core-components/modules.md

```diff
@@ -57,6 +64,9 @@ This module does not define specific variables as input, but variable interpolat
                     "type": "ssh",
                     "command": "ls -la"
                 }
+            ],
+            "out_args": [
```

Comment:
The agreed-upon, upcoming change in CACAO v2.1 (or v3), is to set out_args as a common property of the command. So if we want to anticipate this very reasonable change, we should set out_args within a command in commands. This also changes the description that we would have to provide

Reply:
This comment applies to the rest of this document